src/app.jsx

```diff
@@ -229,17 +220,15 @@ class Application extends React.Component {
                             [system ? "systemContainersLoaded" : "userContainersLoaded"]: true,
                         };
                     });
-                    if (init) {
-                        this.updateContainerStats(system);
-                        for (const container of reply || []) {
-                            this.inspectContainerDetail(container.Id, system);
-                        }
+                    this.updateContainerStats(system);
+                    for (const container of reply || []) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1337-20230705-163654-8d4bd8af-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
I finally understand what this means. I have some follow-up fixes queued up to drop the unnecessary `|| []`.